src/osha/oira/client/browser/client.py

```diff
@@ -182,30 +183,7 @@ def results(self):
 
             brains = catalog(**query)
 
-            sectors = api.portal.get().sectors
-
-            def filter_items(brain):
-                obj = brain.getObject()
-
-                if getattr(obj, "preview", False) or getattr(obj, "obsolete", False):
```

Comment:
These two lines are not ported. A user can subscribe to a preview and an obsolete tool, so it doesn't make sense to exclude them here. And even if it did, the check would have to be made on the counterpart, not the client object.